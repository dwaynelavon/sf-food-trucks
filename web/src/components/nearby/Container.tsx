import React from "react";
import { IReactStandardProps } from "./types";

const Container = ({ children }: IReactStandardProps) => (
    <div className="container box">
        <h4 className="title is-4">Nearby Search</h4>
        {children}
    </div>
);

export default Container;
