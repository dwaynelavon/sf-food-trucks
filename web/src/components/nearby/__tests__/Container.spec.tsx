import { shallow } from "enzyme";
import React from "react";
import Container from "../Container";

describe("Container", () => {
    it("should render with default props", () => {
        const wrapper = shallow(<Container />);
        expect(wrapper).toMatchSnapshot();
    });

    it("should render with children", () => {
        const wrapper = shallow(
            <Container>
                <div>Hello World</div>
            </Container>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
