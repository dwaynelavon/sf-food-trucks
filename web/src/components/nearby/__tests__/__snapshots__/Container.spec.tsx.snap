// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render with children 1`] = `ShallowWrapper {}`;

exports[`Container should render with default props 1`] = `ShallowWrapper {}`;
